### PR TITLE
add exercise 12 files

### PR DESCRIPTION
**Pull request by Cainday-Bradley**

### CODE REVIEW

File: cainday_bradley/script.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+let form = document.getElementById("comment_form");
+
+form.addEventListener("submit", function(event) {
+  event.preventDefault();
```
Comment (ternalapriljoy) [inline cainday_bradley/script.js:4]: form.addEventListener("submit", function(event) {
    
  event.preventDefault();

File: cainday_bradley/style.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  text-align: center;
+  border: 2px solid black;
+  padding: 20px;
+  background-color: #ff748c;
+  margin-bottom: 40px; 
+  width: 48%; 
+  line-height: 1.5;
+}
+
+
```
Comment (ternalapriljoy) [inline cainday_bradley/style.css:121]: Remove space. CSS block should be separated by a single space.

File: cainday_bradley/style.css [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+   text-align: center;
+   border: 2px solid black;
+   background-color: #ffeff2;
+   padding: 15px;
+   width: 50%;
+   margin: 0 auto;
+   margin-bottom: 20px;
+}
+
+
```
Comment (ternalapriljoy) [inline cainday_bradley/style.css:148]: Remove space.

File: cainday_bradley/style.css [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  border: 2px solid black;
+  padding: 20px;
+  background-color: #ff748c;
+  margin-bottom: 40px; 
+  width: 48%; 
+  line-height: 1.5;
+}
+
+
+.gallery-section img {
```
Comment (Ternal-AprilJoy) [inline cainday_bradley/style.css]:
```suggestion

.gallery-section img {
```

File: cainday_bradley/style.css [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+.goals {
+    border: 2px solid black;
+    display: grid;
+    grid-template-areas:
+    "itm1 itm2";
+}
+
+
+h4 {
```
Comment (Ternal-AprilJoy) [inline cainday_bradley/style.css:136]:
```suggestion

h4 {
```
Remove space. CSS block should be seperated by a single space only.

File: cainday_bradley/style.css [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+   border: 2px solid black;
+   background-color: #ffeff2;
+   padding: 15px;
+   width: 50%;
+   margin: 0 auto;
+   margin-bottom: 20px;
+}
+
+
+.my-goals {
```
Comment (Ternal-AprilJoy) [inline cainday_bradley/style.css]:
```suggestion
   margin-bottom: 20px;
}

.my-goals {
```
Remove space.

**Pull request merged**